src/popup.html

```diff
@@ -1,170 +1,304 @@
+      <meta name="viewport" content="width=device-width, initial-scale=1">
+    </head>
+
+    <body class="fx-relay-panel fx-relay">
+      <panel-content>
+        <fx-relay-logo-wrapper>
+          <fx-relay-logomark></fx-relay-logomark>
+          <fx-relay-logotype></fx-relay-logotype>
+          <button class="open-settings settings-toggle i18n-attribute" data-i18n-attribute="title"
+            data-i18n-message-id="popupSettingsTitle"></button>
+        </fx-relay-logo-wrapper>
+
+        <settings>
+          <div class="settings-header">
+            <button class="close-settings settings-toggle i18n-attribute" data-i18n-attribute="title"
+              data-i18n-message-id="popupSettingsReturn"></button>
+            <h1 class="settings-hl i18n-content" data-i18n-message-id="popupSettingsTitle"></h1>
+          </div>
+          <ul class="settings-list">
+            <!-- Settings Toggle -->
+            <li class="setting">
+              <span class="setting-label flex-col i18n-content" data-i18n-message-id="popupSettingsShowIcon"></span>
+              <button title="" data-icon-visibility-option="disable-input-icon"
+                class="toggle-icon-in-page-visibility fx-relay-toggle"></button>
+            </li>
+            <li class="setting">
+              <span class="setting-label flex-col i18n-content" data-i18n-message-id="popupSettingsCollectData"></span>
+              <button title="" class="data-collection fx-relay-toggle"></button>
+            </li>
+            <!-- Settings Redirect -->
+            <li class="setting flex-col">
+              <a id="popupSettingsLeaveFeedbackLink"
+                href="https://addons.mozilla.org/firefox/addon/private-relay/?utm_source=fx-relay-addon&utm_medium=popup"
+                class="setting-label setting-link close-popup-after-click i18n-content" data-event-action="click"
+                data-event-label="panel-leave-feedback-link" data-i18n-message-id="popupSettingsLeaveFeedback"></a>
+              <a href="https://www.mozilla.org/privacy/firefox-relay/?utm_source=fx-relay-addon&utm_medium=popup"
+                class="setting-label setting-link close-popup-after-click i18n-content" data-event-action="click"
+                data-event-label="panel-privacy-link" data-i18n-message-id="popupSettingsPrivacy"></a>
+              <a href="https://www.mozilla.org/about/legal/terms/firefox-relay/?utm_source=fx-relay-addon&utm_medium=popup"
+                class="setting-label setting-link close-popup-after-click i18n-content" data-event-action="click"
+                data-event-label="panel-terms-of-service" data-i18n-message-id="popupSettingsTermsOfService"></a>
+              <!-- Report Issue Panel Prompt -->
+              <a class="setting-label setting-link settings-report-issue i18n-content" data-event-action="click"
+                data-event-label="panel-report-issue" data-i18n-message-id="popupSettingsReportIssue"></a>
```

Comment:
continued from previous comment above: this snippet should also be included on the file reset.